src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -1392,7 +1392,7 @@ static void ValidateLength(string decoratorName, DecoratorSyntax decoratorSyntax
 
```

Comment:
I assume the next two changes are what caused "//@[03:09) [BCP297 (Error)] Function "secure" cannot be used as a type decorator. (CodeDescription: none) |secure|" to disappear?

Perhaps that should be in a separate check so that we get a better expert on that area to see it, and also because it might need more tests.